docs/Introduction.GettingStarted.md

```diff
@@ -39,7 +39,10 @@ Node is the JavaScript runtime Detox will run on. **Install Node 8.3.0 or above*
 
 A collection of utils for Apple simulators, Detox uses it to communicate with the simulator.
 
+If you are using `MacOS Mojave` run `xcode-select install` first
```

Comment:
Let’s just put the line in the instructions. In April, Apple will require Xcode 11, which is Mojave and above only.